coffee-chats/src/main/webapp/src/util/stringUtils.ts

```diff
@@ -0,0 +1,5 @@
+export const capitaliseEachWord = (input: string) => {
+  return input.split(' ')
```

Comment:
How does this work with unicode?

Reply:
So this should work with unicode characters, however it does not account for language-specific edge-cases to do with capitalisation. Without using a library, I am unsure what would be the best way to implement this.